site/app/templates/admin/users/ToggleManageStudentsColumns.twig

```diff
@@ -64,6 +64,6 @@
 
 {% block buttons %}
     {{ block('close_button') }}
-    <input name="submit" class="btn btn-primary key_to_click" type="submit" value="Submit" onclick="updateManageStudentsColumns()"/>
+    <input name="submit" class="btn btn-primary key_to_click" type="submit" value="Submit" id="submit" onclick="updateManageStudentsColumns()"/>
```

Comment:
Same comment here.  This identifier needs to be more specific.

Reply:
done